### PR TITLE
make images of contributors smaller and lazy-loaded

### LINKED ISSUE #412: Improve performances
### Feature Request

|    Q        |   A
|------------ | ------
| New Feature | no
| RFC         | yes
| BC Break    |no

#### Summary

We could improve loading speed in homepage. I see two possible points (maybe other can be proposed):

* use smaller images for contributors
* load such images lazily

### PR DESCRIPTION
**Pull request by garak**
Fix #412

### CODE REVIEW

File: src/Contributors/Model/Contributor.php [hunk 1]
```diff
@@ -10,12 +10,14 @@ class Contributor
     private string $username;
     private string $profileUrl;
     private string $profileImg;
+    private int $size;
 
-    private function __construct(string $username, string $profileUrl, string $profileImg)
+    private function __construct(string $username, string $profileUrl, string $profileImg, int $size = 160)
```
[the remark this answers is not in the document]
Comment (garak) [reply src/Contributors/Model/Contributor.php:15]: But a constant can't be changed

[the remark this answers is not in the document]
Comment (AlessandroMinoccheri) [reply src/Contributors/Model/Contributor.php:15]: I mean instead of 160.

**Review by JellyBellyDev — APPROVED** [review]
LGTM

**Review by AlessandroMinoccheri — APPROVED** [review]
LGTM 👍

**Pull request merged**